src/Microsoft.Azure.WebJobs.Host/Executors/JobHostContext.cs

```diff
@@ -17,20 +18,26 @@ internal sealed class JobHostContext : IDisposable
         private readonly IListener _listener;
         private readonly TraceWriter _trace;
         private readonly IAsyncCollector<FunctionInstanceLogEntry> _fastLogger; // optional
+        private readonly IAsyncCollector<FunctionResultLog> _aggregator;
```

Comment:
But both of these are just publish endpoints from the runtime (and then Script or somebody else consumes it); and they're publishing semantically same information.  
So we're not forcing a choice. 



---
In reply to: [112053494](https://github.com/Azure/azure-webjobs-sdk/pull/1059#discussion_r112053494) [](ancestors = 112053494)